### PR TITLE
validated input field in my customer

### PR DESCRIPTION
**Pull request by Hopekodi**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/customer/create.blade.php [hunk 1]
```diff
@@ -51,26 +51,26 @@
...
+                                <input class="form-control" type="text" value="John Doe" required name="name">
                               </div>
                             </div>
 
                             <div class="form-group">
                               <label class="col-lg-3 control-label">Email:</label>
                               <div class="col-lg-8">
-                                <input class="form-control" type="text" value="johndoe@doetech.com">
+                                <input class="form-control" type="text" value="johndoe@doetech.com" name="email" required>
                               </div>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php:62]: why does this have a default value?

File: resources/views/backend/customer/create.blade.php [hunk 2]
```diff
@@ -51,26 +51,26 @@
                             <div class="form-group">
                               <label class="col-lg-3 control-label">Full Name:</label>
                               <div class="col-lg-8">
-                                <input class="form-control" type="text" value="John Doe">
+                                <input class="form-control" type="text" value="John Doe" required name="name">
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php:54]: remove default value

File: resources/views/backend/customer/create.blade.php [hunk 3]
```diff
@@ -51,26 +51,26 @@
...
+                                <input class="form-control" type="text" value="johndoe@doetech.com" name="email" required>
                               </div>
                             </div>
 
                             <div class="form-group">
                                 <label class="col-lg-3 control-label">Tel:</label>
                                 <div class="col-lg-8">
-                                  <input class="form-control" type="phone" value="+234 90 00 000 000">
+                                  <input class="form-control" type="phone" value="+234 90 00 000 000" name="phone_number" required>
                                 </div>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php:69]: same thing here

File: resources/views/backend/customer/create.blade.php [hunk 4]
```diff
@@ -51,26 +51,26 @@
...
                               </div>
                               <div class="form-group">
                                 <div class="col-md-8">
-                                    <select class="form-control">
+                                    <select class="form-control" required>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php:73]: this does not have a name attr

File: resources/views/backend/customer/create.blade.php [hunk 5]
```diff
@@ -80,7 +80,7 @@
                               </div>  
                               <div class="form-group">
                                 <div class="col-md-8">
-                                    <select class="form-control">
+                                    <select class="form-control" required>
                                         <option selected="">Status</option>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php:84]: no name attr

**Comment by Hopekodi** [comment]
That's not my task, I did my task and asked dem whether i should push and they said yes,maybe I should just wait till we finish everything

File: resources/views/backend/customer/create.blade.php [hunk 6]
```diff
@@ -51,26 +51,26 @@
                             <div class="form-group">
                               <label class="col-lg-3 control-label">Full Name:</label>
                               <div class="col-lg-8">
-                                <input class="form-control" type="text" value="John Doe">
+                                <input class="form-control" type="text" maxlength="30" value="John Doe" required name="name">
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php]: you are putting value for create
It should be value="{{ old('name') }}

File: resources/views/backend/customer/create.blade.php [hunk 7]
```diff
@@ -51,26 +51,26 @@
...
                               </div>
                             </div>
 
                             <div class="form-group">
                               <label class="col-lg-3 control-label">Email:</label>
                               <div class="col-lg-8">
-                                <input class="form-control" type="text" value="johndoe@doetech.com">
+                                <input class="form-control" type="text" value="johndoe@doetech.com" name="email" required>
```
Comment (Jeremiahiro) [inline resources/views/backend/customer/create.blade.php]: you might want to set a placeholder instead of using dummy data as value

**Pull request merged**